src/test/statistics/chunk_statistics/histograms/equal_width_histogram_test.cpp

```diff
@@ -507,6 +507,24 @@ TEST_F(EqualWidthHistogramTest, FloatLessThan) {
                   4.f + 7.f + 3.f);
 }
 
+TEST_F(EqualWidthHistogramTest, FloatRounding) {
+  // Values are adapted from an actual error.
+  const auto min = 1023.79f;
+  const auto max = 694486.f;
+  const auto bin_count = 10000u;
+  const auto hist = EqualWidthHistogram<float>(min, max, std::vector<HistogramCountType>(bin_count, 1u),
+                                               std::vector<HistogramCountType>(bin_count, 1u), 0u);
+  EXPECT_NO_THROW(hist.estimate_cardinality(PredicateCondition::GreaterThanEquals, max));
```

Comment:
That's not what I meant. If we think we should test for NOTHROW, there needs to be a very good explanation, because this test could be done for every single function in Hyrise.
I would see the previous behaviour as a bug that is hard to find out. If we want to have a test for that bug, there should be a comment explaining that.

Reply:
Sorry for the delay, please check whether the comments make sense to you.